trustfall_stubgen/src/util.rs

```diff
@@ -138,3 +138,11 @@ pub(crate) fn field_value_to_rust_type(
         #base.#suffix
     }
 }
+
+pub fn escaped_rust_name(name: String) -> String {
```

Comment:
The result of this function is always used by reference, so let's make this take `&str` and have it return `Cow<'_, str>` to avoid some unnecessary cloning:
https://doc.rust-lang.org/std/borrow/enum.Cow.html

Reply:
It's no longer only used by reference, see: [25f2dee](https://github.com/obi1kenobi/trustfall/pull/406/commits/25f2dee40af223e6d87defd703b82f423149949e) that stores the String's by value in the map, so it's much easier to keep the String return type